src/libraries/System.Data.Common/src/System/Xml/DataPointer.cs

```diff
@@ -219,14 +216,14 @@ internal bool MoveToNextSibling()
                         return false;
                     }
 
-                    DataColumn c = NextColumn(Row, _column, false, false);
+                    DataColumn? c = NextColumn(Row, _column, false, false);
                     if (c != null)
                     {
                         MoveTo(_node, c, false);
                         return true;
                     }
 
-                    XmlNode n = _doc.SafeFirstChild(_node);
+                    XmlNode? n = _doc.SafeFirstChild(_node);
```

Comment:
Interesting method name, for something that may return null 😄 .

Reply:
I had the same thought but `Safe` here seems to refer to a sane way to unwrap the nodes.